### ISSUE TITLE
`raw_message_delivery`  field of `aws_sns_topic_subscription` doesn't tracked when modified on AWS's side

### ISSUE THREAD

**Issue by opteemister** [terraform-providers/terraform-provider-aws#1910]
### Terraform Version
Terraform v0.10.7
provider "aws" (1.1.0)

### Affected Resource(s)
aws_sns_topic_subscription

### Terraform Configuration Files

```hcl
resource "aws_sns_topic_subscription" "sns_topic_subscription" {
  count = "${var.sns_enabled == "true" ? 1 : 0 }"
  topic_arn = "${var.sns_topic}"
  protocol  = "sqs"
  endpoint  = "${aws_sqs_queue.queue.arn}"
  raw_message_delivery = "true
}
```

### Steps to Reproduce
1. `terraform apply`
2. Change `raw_message_delivery` parameter for newly created resource `sns_subscription`. From `true` to `false` on AWS's side.
3. `terraform plan` 

### Expected Behavior
`aws_sns_topic_subscription.sns_topic_subscription.raw_message_delivery` should be marked as changed and add `false` to `true` change in plan.

### Actual Behavior
no changes tracked for field `aws_sns_topic_subscription.sns_topic_subscription.raw_message_delivery`

**Comment by maroux** [comment]
I'm seeing the same thing - TF didn't apply the value for `raw_message_delivery` correctly on version 0.11

**Comment by DimDroll** [comment]
Reporting same issue here:
```
resource "aws_sns_topic_subscription" "sns_to_sqs_lambda" {
  endpoint             = "${local.s3_lambda_s3_to_sqs_arn}"
  protocol             = "lambda"
  topic_arn            = "${local.source_bucket_sns_topic_arn}"
  raw_message_delivery = true
}
```
Apply:
```
aws_sns_topic_subscription.sns_to_sqs_lambda: Creating...
  arn:                             "" => "<computed>"
  confirmation_timeout_in_minutes: "" => "1"
  endpoint:                        "" => "<redacted>"
  endpoint_auto_confirms:          "" => "false"
  protocol:                        "" => "lambda"
  raw_message_delivery:            "" => "true"
  topic_arn:                       "" => "<redacted>"
```
But it deployed as false:
```
{
  "Attributes": {
    "Endpoint": "<redacted>",
    "Owner": "<redacted>",
    "PendingConfirmation": "false",
    "SubscriptionArn": "<redacted>",
    "ConfirmationWasAuthenticated": "true",
    "RawMessageDelivery": "false",
    "TopicArn": "<redacted>",
    "Protocol": "lambda"
  }
}
```

**Comment by bflad** [comment]
Pull request submitted: #6023

**Comment by bflad** [comment]
The fix for the original issue has been merged and will release with version 1.39.0 of the AWS provider, likely later today.
